pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -565,15 +574,17 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 						forwardPort.Protocol = "TCP"
 					}
 
-					if _, ok := portForwardMap[fmt.Sprintf("%s-%d", forwardPort.Protocol, forwardPort.Port)]; ok {
-						causes = append(causes, metav1.StatusCause{
-							Type:    metav1.CauseTypeFieldValueInvalid,
-							Message: fmt.Sprintf("Port and protocol combination must be unique"),
-							Field:   field.Child("domain", "devices", "interfaces").Index(idx).Child("ports").Index(portIdx).String(),
```

Comment:
You no longer validate that there are no duplicate port definitions for the same interface. Was the change intended?